tests/test_deleting.py

```diff
@@ -266,6 +266,29 @@ def assert_deletion(was_deleted=False, **kw):
         response = self.app.delete('/api/person/1')
         assert response.status_code == 204
 
+    def test_postprocessor_no_commit_on_error(self):
+        """Tests that an Exception in a postprocessor ensures that the session
+        is not getting commited but just flushed and will be rolled back once closed."""
+
+        def raise_error(**kw):
+            raise ProcessingException(status=500)
+
+        person = self.Person(id=1)
+        self.session.add(person)
+        self.session.commit()
+
+        postprocessors = dict(DELETE_RESOURCE=[raise_error])
+        self.manager.create_api(self.Person, methods=['DELETE'],
+                                postprocessors=postprocessors)
+        response = self.app.delete('/api/person/1')
+
+        assert response.status_code == 500
+        person_count = self.session.query(self.Person).count()
+        assert person_count == 0
+        self.session.rollback()
+        person_count = self.session.query(self.Person).count()
```

Comment:
A better check here would be 

    people = self.session.query(self.Person).all()
    assert people == [person]

since this asserts that the person we originally created remains in the database.